packages/mobile/src/account/reducer.ts

```diff
@@ -30,6 +30,7 @@ export interface State {
   acceptedTerms: boolean
   hasMigratedToNewBip39: boolean
   choseToRestoreAccount: boolean | undefined
+  profileUploaded: boolean
```

Comment:
If we don't have a redux state migration:
```suggestion
  profileUploaded: boolean | undefined
```